tslint.json

```diff
@@ -0,0 +1,318 @@
+            "parameters",
+            "arguments",
+            "statements"
+        ],
+        // TODO: "curly": true,
+        // TODO: "eofline": true,
+        "import-spacing": true,
+        "indent": [
+            false, // TODO: true,
+            "spaces"
+        ],
+        "linebreak-style": false, //changed
+        "newline-before-return": false, // changed
+        // TODO: "no-consecutive-blank-lines": true,
+        "no-empty-line-after-opening-brace": false,
+        // TODO: "no-single-line-block-comment": true,
+        "no-trailing-whitespace": true,
+        "no-unnecessary-semicolons": true,
+        "object-literal-key-quotes": [
+            false, // TODO: true,
+            "as-needed"
+        ],
+        "one-line": [
+            false, // TODO: true,
+            "check-open-brace",
+            "check-catch",
+            "check-else",
+            "check-whitespace"
+        ],
+        "quotemark": [
+            false, // changed
+            "single"
+        ],
+        "react-tsx-curly-spacing": true,
+        "semicolon": [
+            false, // TODO: true,
+            "always"
+        ],
+        "trailing-comma": [
+            false, // TODO: true,
+            {
+                "singleline": "never",
+                "multiline": "never"
+            }
+        ],
+        "typedef-whitespace": false,
+        "whitespace": [
+            true,
+            "check-branch",
+            "check-decl",
+            "check-operator",
+            "check-separator",
+            "check-type"
+        ],
+        "ban": false,
+        "ban-types": true,
+        "cyclomatic-complexity": true,
+        "file-header": false,
+        "import-blacklist": false,
+        "interface-over-type-literal": false,
+        "no-angle-bracket-type-assertion": false,
+        "no-inferred-empty-object-type": false,
+        "no-internal-module": false,
+        "no-magic-numbers": false,
+        "no-mergeable-namespace": false,
+        "no-namespace": false,
+        "no-reference": true,
+        "no-unexternalized-strings": [
+            false,
+            {
+                "signatures": [
+                    "localize",
+                    "nls.localize"
+                ],
+                "keyIndex": 0,
+                "messageIndex": 1
+            }
+        ],
+        "object-literal-shorthand": false,
+        "prefer-type-cast": true,
+        "space-before-function-paren": false,
+        "missing-optional-annotation": false,
+        "no-duplicate-parameter-names": false,
+        "no-empty-interfaces": false,
+        "no-missing-visibility-modifiers": false,
+        "no-multiple-var-decl": false,
+        "no-switch-case-fall-through": false,
+        "typeof-compare": false
+    },
+    "rulesDirectory": "node_modules/tslint-microsoft-contrib/",
+    "linterOptions": {
+        "exclude": [
+            "src/diagnostics/**", // should be handled by the Log Points team
```

Comment:
Remove these exclusions - I believe they were only applicable to the app service repo